### ISSUE TITLE
Suggested Announced Content for "How to document the screen reader user experience" has a non-compliance with 2.5.3. Label in Name

### ISSUE THREAD

**Issue by emleml** [bbc/accessibility-news-and-you#73]
Hello,
I just wanted to first preface this issue with a really big thank you for this fantastic article.

https://bbc.github.io/accessibility-news-and-you/guides/screen-reader-ux.html

Within your suggested Announced Content for the proposed elements, #8, Copy Link is translated to Copy article link. Unfortunately this is a failure against https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html, more specifically the failure https://www.w3.org/WAI/WCAG21/Techniques/failures/F96 , Example 2 "Invisible link text disrupts visible label text string in accessible name". Technically this discusses interspersed hidden text, but I believe a replacement with aria-label would also cause this to be a failure, as is another proposed failure "@@ Accessible name contains the visible label text, but one or more other words are interspersed in the label".

This would not fail if the accessible name was instead "Copy link to article" or equivalent.

If there is interspersed text, somebody navigating with speech navigation software may ask to go to the Copy link button, but due to the word article being within the string it may not be picked up.

Thank you again for the very insightful article!
